src/components/TokenLocking/ActivityRewardsInfo.tsx

```diff
@@ -85,12 +88,10 @@ export const ActivityRewardsInfo = () => {
               Repeat!
             </Typography>
           </div>
-
           <Divider />
-
-          <Link href="#" sx={{ textAlign: 'center', fontSize: '14px' }}>
+          <NextLink style={{ textAlign: 'center', fontSize: '14px' }} href={AppRoutes.activityProgram}>
```

Comment:
We have to use a MUI Link which uses the NextLink as component:

```ts
<Link href={AppRoutes.activityProgram} component={NextLink} sx={{ textAlign: 'center', fontSize: '14px' }}>
            View eligible activities
</Link>
```

Otherwise it will not have our theme applied to it